exercises/rna-transcription/example/RNA.ceylon

```diff
@@ -2,12 +2,12 @@ class Error() {}
 
 String|Error transcription(String dna) {
   return dna.fold("" of String|Error)((String|Error r,
-                                       Character    c) =>   if(is Error r)
-                                                          then(r)
-                                                          else(switch(c)
-                                                               case('G') r + "C"
-                                                               case('C') r + "G"
-                                                               case('T') r + "A"
-                                                               case('A') r + "U"
-                                                               else(Error())));
+                                       Character    c) =>   if (is Error r)
+                                                          then (r)
+                                                          else (switch(c)
```

Comment:
Applicable to all these six lines (the `switch`, `case`, `else`) - I guess now I wonder whether there should be spaces between the keyword and the following parenthesis for these as well. It looks a little inconsistent that I asked for spaces after the `if` and `then` and the `else` on this line, but not for these six lines.

I just checked and found that https://github.com/ceylon/ceylon.formatter exists; maybe it should be used instead of me making up rules since it's not like I know what I'm doing.

I'll take a look to see whether it's usable. If it is not usable, I am starting to think it will be a bit difficult to enforce consistency across the entire project, though consistency within one file is most certainly good. I will be happy with consistently a space or consistently no space in this file.

Reply:
Yeah, I'd probably want the spaces if there's going to be spaces for the `if`, `then`, and `else`. Consistency is pleasing.